cmd/gen-docs/main.go

```diff
@@ -41,9 +45,13 @@ func run(args []string) error {
 	}
 
 	ios, _, _, _ := iostreams.Test()
-	rootCmd := root.NewCmdRoot(&cmdutil.Factory{
+	rootCmd, _ := root.NewCmdRoot(&cmdutil.Factory{
 		IOStreams: ios,
 		Browser:   &browser{},
+		Config: func() (config.Config, error) {
+			return config.NewBlankConfig(), nil
+		},
+		ExtensionManager: &em{},
```

Comment:
`NewCmdRoot` now relies on `Config` and `ExtensionManager`, so I stubbed them out here.